consensus/XDPoS/engines/engine_v1/engine.go

```diff
@@ -78,772 +66,95 @@ panics. This is done to avoid accidentally using both forms (signature present
 or not), which could be abused to produce different hashes for the same header.
 */
 func (x *XDPoS_v1) SigHash(header *types.Header) (hash common.Hash) {
-	return sigHash(header)
+	return common.Hash{}
```

Comment:
Why do we still need this file? shouldn't we just remove the whole v1 directory?